### PR TITLE
Add CLI parameter if value is defined or 0

### PR DESCRIPTION
**Pull request by thibauthourlier**
This fixes the problem with the meta table where the meta key
repeat.analysis is wrong for repeatmasker. The problem comes from
parameters being added to the command line even if the value is empty
string.
Now the value needs to be "true" or 0

# Requirements
When creating your Pull request, please fill out the template below:

# PR details
_Is this a fix/ update/ new feature?_
fix

_Include a short description_
This fixes the problem with the meta table where the meta key
repeat.analysis is wrong for repeatmasker. The problem comes from
parameters being added to the command line even if the value is empty
string.
Now the value needs to be "true" or 0

_Include links to JIRA tickets_

# Testing
_Have you tested it?_
No, but it is trivial enough. However only a full run of the pipeline would fully test any possible problem

# Assign to the weekly GitHub reviewer
_If you are a member of Ensembl, please check the Genebuild weekly Rotas and assign this week's GitHub reviewer to the PR_

### CODE REVIEW

**Comment by thibauthourlier** [comment]
@leannehaggerty you are a reviewer only to be able to know when the repeat meta key should be fixed

**Review by EreboPSilva — APPROVED** [review]
Looks good to me. The fix match what the description says it should do.
I approve it and wait for approval of @ens-ftricomi.

**Review by ens-ftricomi — COMMENTED** [review]
That's perfect, thanks a lot Thibaut

**Pull request merged**
